tf-modules/cumulus/variables.tf

```diff
@@ -170,7 +170,7 @@ variable "ecs_docker_hub_config" {
 
 variable "ecs_docker_storage_driver" {
   type    = string
-  default = "overlay2"
```

Comment:
Shouldn't you reproduce this in `packages/deployment/app/config.yml` as well?

Reply:
Updated